### PR TITLE
Add new --labels options to nunitlite

### LINKED ISSUE #1830: Add --labels switch changes to nunilite and nunitlite tests
As per @CharliePoole, I am opening this issue to make sure that the --label changes (issue24 nunit/nunit-console#78) made to the console are also made to nunitlite and its tests.

### PR DESCRIPTION
**Pull request by CharliePoole**
Fixes #1830 

* Adds Before and After options.
* Corrects operation of All option.
* Adds extensive tests modeled on the console tests (implementation is different, of course).
* Corrects display of Write versus WriteLine, eliminating newlines except where absolutely needed.

### CODE REVIEW

File: src/NUnitFramework/nunitlite/TextUI.cs [hunk 1]
```diff
@@ -40,23 +40,36 @@ public class TextUI
...
+#endif
+            _displayBeforeTest = labelsOption == "ALL" || labelsOption == "BEFORE";
+            _displayAfterTest = labelsOption == "AFTER";
+            _displayBeforeOutput = _displayBeforeTest || _displayAfterTest || labelsOption == "ON";
         }
 
         #endregion
 
-        #region Public Methods
+#region Public Methods
```
Comment (rprouse) [inline src/NUnitFramework/nunitlite/TextUI.cs]: I really wish they would fix this in Visual Studio 😦

File: src/NUnitFramework/nunitlite/TextUI.cs [hunk 2]
```diff
@@ -40,23 +40,36 @@ public class TextUI
...
+        #region Constructor
 
         public TextUI(ExtendedTextWriter writer, TextReader reader, NUnitLiteOptions options)
         {
             Writer = writer;
             _reader = reader;
             _options = options;
+
+#if NETSTANDARD1_6 || PORTABLE
+            string labelsOption = options.DisplayTestLabels?.ToUpper() ?? "ON";
```
Comment (rprouse) [inline src/NUnitFramework/nunitlite/TextUI.cs]: You could use `ToUpperInvariant()` for all platforms losing the `#if`

File: src/NUnitFramework/nunitlite/TextUI.cs (hunk 1 above)
Comment (CharliePoole) [inline src/NUnitFramework/nunitlite/TextUI.cs]: I just lined them all up again but maybe we should just acquiesce to keeping them where VS seems to want them. This file probably has too many regions.

**Comment by rprouse** [comment]
Looks good, just waiting for CI to pass.

**Comment by oznetmaster** [comment]
I have commented about PR on the commit

**Pull request merged**